src/Components/Form/FormFields/TextFormField.tsx

```diff
@@ -61,7 +61,7 @@ const TextFormField = forwardRef((props: TextFormFieldProps, ref) => {
       required={field.required}
       onFocus={props.onFocus}
       onBlur={props.onBlur}
-      onChange={(e) => field.handleChange(e.target.value)}
+      onChange={props.onChange}
```

Comment:
Why was this changed?

This would break all of the other existing usages of the text form field in the platform.